src/operator/nn/concat.cc

```diff
@@ -345,25 +345,29 @@ The storage type of ``concat`` output depends on storage types of inputs
 
 Example::
 
-   x = [[1,1],[2,2]]
-   y = [[3,3],[4,4],[5,5]]
-   z = [[6,6], [7,7],[8,8]]
-
-   concat(x,y,z,dim=0) = [[ 1.,  1.],
-                          [ 2.,  2.],
-                          [ 3.,  3.],
-                          [ 4.,  4.],
-                          [ 5.,  5.],
-                          [ 6.,  6.],
-                          [ 7.,  7.],
-                          [ 8.,  8.]]
```

Comment:
Have the community decided to remove all language-agnostic pseudo code example? Other language binding may loose the only available example